### DISCUSSION TITLE
New code edit feature coming soon - Would you find this helpful?

### DISCUSSION THREAD

**Discussion by jackMort** [jackMort/ChatGPT.nvim#15] [Announcements]
Hello! 

I'm currently working on adding a new feature to the ChatGPT.nvim plugin that allows for code edits. I'm wondering if this feature would be helpful for you in your workflow. 

[![Alt text](https://img.youtube.com/vi/dWe01EV0q3Q/0.jpg)](https://www.youtube.com/watch?v=dWe01EV0q3Q)

Please let me know your thoughts on this feature and if you have any suggestions for how it could be improved. I value your feedback and want to make sure I'm building a product that meets your needs.

Thanks in advance for your input!

**Comment by ADIX7** [comment]
Is there any way to show the text answer too? For example I just asked chatgpt to fix me code but it didn't have any error already, so I just get back the same code, but I didn't get back the text that it is already correct. The simple texts can have some relevant information too.
